packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -19,32 +19,30 @@ class CpuProfilerController {
   static CpuProfileData baseStateCpuProfileData = CpuProfileData.empty();
 
   /// Notifies that new cpu profile data is available.
-  ValueListenable get dataNotifier => _dataNotifier;
+  ValueListenable get data => _dataNotifier;
```

Comment:
this refactor looks only half done. The privates still have _notifier in the name. 
Also looks unrelated. Maybe move to a separate CL?